rocketpy/Flight.py

```diff
@@ -1343,7 +1343,11 @@ def uDot(self, t, u, postProcessing=False):
             if aerodynamicSurface[-1] == "Fins":
                 Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface[2]
                 Clf = Clfdelta * cantAngleRad
-                Cld = Cldomega * omega3 / freestreamSpeed if freestreamSpeed != 0 else 0
+                Cld = (
+                        Cldomega * omega3 * min(1, 1 / freestreamSpeed)
+                        if freestreamSpeed != 0
+                        else Cldomega * omega3
+                    )
```

Comment:
```suggestion
                Cld = Cldomega * np.arctan2(omega3, freestreamSpeed)
```

Reply:
Perhaps using [np.arctan2](https://numpy.org/doc/stable/reference/generated/numpy.arctan2.html) can help us solve this. In case the freestream speed is zero, it returns 0 if oemga3 is also zero, otherwise it will return at most pi/2.

This does not account for stall, but closely follows the linear relationship before stall occurs and does not explode when stall occurs.

![image](https://user-images.githubusercontent.com/572483/150899993-83aeb866-d497-4a5f-86ba-8dbcc698ca1d.png)